descartes_light/test/src/perf_plotter.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env python
+
+import math
+import numpy as np
+# import matplotlib as mpl
+import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d import axes3d, Axes3D #<-- Note the capitalization! 
+import sys
+
+class Perf_Plotter():
```

Comment:
Please add Python-style doc strings to the class and function members to describe their purpose and inputs. It's important to document Python code thoroughly (including type names of function arguments) so we can understand what's happening here in the future

```python
class Perf_Plotter():
""" Class for plotting the results of solver benchmarks
"""

def strip(self, filename):
""" Parses the input file and extracts benchmark information

:param filename: (str) filename for the file containing benchmark information
:return: (dict) a dictionary of benchmark information organized in the following way ...
"""
```